perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_def_vomsGroupName.java

```diff
@@ -28,9 +28,6 @@
 	public void checkAttributeValue(PerunSessionImpl perunSession, Group group, Attribute attribute) throws InternalErrorException, WrongAttributeValueException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException {
 		String vomsGroupName;
 
-		if(attribute.getValue() == null) {
```

Comment:
Please make it ``if (attribute.getValue() == null) return;``. I'm not sure, that ``matcher.matches()`` would work on null value.